<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>21</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_newGame">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>New Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_startGame">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_endGame">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>End Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_Save">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_quitGame">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>350</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_Undo">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_Reroll">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Re-roll</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_roll">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rollValue">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_currPlayer">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Player: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_timeLeft">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Left: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rollValue_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll Value:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_undoCount">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Undos Left: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rerollCount">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>310</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rerolls Left: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_acceptMove">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>350</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Accept Move</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
